datreant/__init__.py

```diff
@@ -12,15 +12,12 @@
 
 """
 # Bring some often used objects into the current namespace
-from datreant.treants import Treant, Group, register
-from datreant.coordinator import Coordinator
-from datreant.collections import Bundle
-from datreant.manipulators import *
-import datreant.backends
-import datreant.limbs
-import datreant.data
+from .treants import Treant, Group, register
+from .collections import Bundle
+from .manipulators import *
```

Comment:
Agreed. Removing `manipulators` as an import for now. There's only the one (`datreant.manipulators.get()`), and we should explicitly include these anyhow.